dwave/preprocessing/composites/clip.py

```diff
@@ -27,6 +27,9 @@ class ClipComposite(ComposedSampler):
        sampler (:class:`dimod.Sampler`):
             A dimod sampler.
 
+    .. versionadded:: 0.6.8
+        Support for context manager protocol and :meth:`.close` method.
+
```

Comment:
Line 43: `sampler = ClipComposite(ExactSolver())` outside of a context manager and not closed